test/e2e/e2e_suite_test.go

```diff
@@ -116,7 +116,7 @@ func initK8SClient() error {
 	return err
 }
 
-func createNamespace(prefix string) error {
+func createTestingNamespace(prefix string) error {
```

Comment:
I think it should be the other way around. Create the NS and setting the `fixture.TestingNS` in the calling site. Because that seem the only dependency on fixture/testing side of things, otherwise this function is creating a regular namespace.

Reply:
maybe not:
```
func createNamespace(prefix string) (*corev1.Namespace, error) {
    ...
}

...

ns, err := createNamespace("mutualcpu-testing-")
Expect(err).ToNot(HaveOccurred)
fixture.TestingNS = ns
```